freestyle-cache-redis/shared/src/main/scala/rediscala/Kleislis.scala

```diff
@@ -26,6 +23,17 @@ private[rediscala] trait StringCommandsCont {
   def set[Key: Format, Value: Serializer](key: Key, value: Value): Ops[Future, Boolean] =
     Kleisli((client: StringCommands) => client.set(key, value))
 
+  def mset[Key : Format, Value: Serializer](keyValues: Map[Key, Value])(implicit format: Format[Key]): Ops[Future, Boolean] = {
```

Comment:
If you add the implicit parameters `format: Format[Key]`, then you don't need to put the constrain `Key: Format` in the declaration of type parameters.